Baikal/Kernels/CL/fill_aovs_uberv2.cl

```diff
@@ -37,7 +37,7 @@ THE SOFTWARE.
 #include <../Baikal/Kernels/CL/vertex.cl>
 
 #define CORRECT_VALUE(value)\
-    if (any(isnan(value.xyz)) || (value.w == 0))\
+    if (any(isnan(value)) || (value.w == 0))\
```

Comment:
Better remove `;` from the definition and put after macro calls `CORRECT_VALUE(aov_background[idx])` - this is more natural way.

Reply:
BTW, if value argument is a function call, it will be called twice.

Reply:
Done